gammapy/background/fov.py

```diff
@@ -3,3 +3,53 @@
 Field-of-view (FOV) background estimation
 """
 from __future__ import print_function, division
+from astropy.wcs import WCS
+from astropy.wcs.utils import pixel_to_skycoord
+from scipy.interpolate import interp1d
+
+__all__ = ['fill_acceptance_image',
+           ]
+
+
+def fill_acceptance_image(image, center, offset, acceptance):
+    """Generate a 2D image of a radial acceptance curve given as
+    an array of values defined at the specified offsets.
+
+    Parameters
+    ----------
+    image : ImageHDU
+        Empty image to fill.
+    center : SkyCoord
+        Coordinate of the center of the image.
+    offset : `~numpy.ndarray`
```

Comment:
We should use Astropy quantity objects as much as possible.
FYI: this is not the case in Astropy core because most of that code was written before `astropy.units` existed, but the guideline is that new code should use quantities where possible / useful.

In this case `offset` should be an `Angle` object, which is a `Quantity` sub-class.
Example: https://gammapy.readthedocs.org/en/latest/api/gammapy.irf.EnergyDependentTablePSF.html